CONTRIBUTING.md

```diff
@@ -0,0 +1,123 @@
+# Contributing to Command Set Registry
+
+Thanks for taking the time to contribute. We're thankful and will do our best to make progress with you.
+
+> In case this your first time contributing on GitHub, you can watch this [series](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) to learn how to contribute.
+
+We have a [Code of Conduct](CODE_OF_CONDUCT), please take a look before you proceed.
+
+## Contributing to docs
+
+If you found a typo, just make a pull request with the fix. But if you're planning to write docs for a command set or a command, raise an issue and then proceed.
+
+## Requesting for a new feature/command
+
+Raise an issue with a brief description about your problem and in what way a command/feature can solve it.
```

Comment:
we should add bug vs feature report templates, see for example https://github.com/adobe/aio-cli-plugin-runtime/tree/master/.github.

Reply:
#46